assets/prototype/domain/eventEditor/containers/useConfigProvider.ts

```diff
@@ -0,0 +1,25 @@
+import { useContext } from 'react';
+import { pick } from 'ramda';
+import { ConfigContext } from '../../../infrastructure/services/contextProviders/ConfigProvider';
+
+type Props = {
+	dateFormat: boolean;
+	email: boolean;
+	timezone: boolean;
```

Comment:
`boolean`??

Reply:
renamed the type name to make more clear the intention